src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheckTest.java

```diff
@@ -22,9 +22,7 @@
 import static com.google.common.truth.Truth.assertWithMessage;
 import static com.puppycrawl.tools.checkstyle.checks.coding.RequireThisCheck.MSG_METHOD;
 import static com.puppycrawl.tools.checkstyle.checks.coding.RequireThisCheck.MSG_VARIABLE;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Bad change.

Reply:
> Why shouldn't this work go under Issue #8126?

So Should I refer my changes to that PR?

The abstract frame check is used for both RedundantThis and RequireThis and This was the reason of Code coverage as the frames were not involved in Redundant This.

Reply:
Done